script/limber

```diff
@@ -53,7 +54,7 @@ begin
   $stdout.puts 'Building configuration'
   $stdout.puts `bundle exec rake config:generate`
   $stdout.puts 'Running application'
-  exec('bundle exec rails s')
+  exec("bundle exec rails #{command}")
 rescue Errno::ENOENT
```

Comment:
`./script/limber` lets you preconfigure limber to talk to different environments with `./script/limber ENVIRONMENT` this lets you also start up a rails console, via `./script/limber ENVIRONMENT c`